docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -372,10 +372,14 @@ Json Parameters:
 -   **StopSignal** - Signal to stop a container as a string or unsigned integer. `SIGTERM` by default.
 -   **HostConfig**
     -   **Binds** – A list of volume bindings for this container. Each volume binding is a string in one of these forms:
-           + `host_path:container_path` to bind-mount a host path into the container
-           + `host_path:container_path:ro` to make the bind-mount read-only inside the container.
-           + `volume_name:container_path` to bind-mount a volume managed by a volume plugin into the container.
-           + `volume_name:container_path:ro` to make the bind mount read-only inside the container.
+           + `host_path:container_path[:options]` to bind-mount a host path into the container
+           + `volume_name:container_path[:options]` to bind-mount a volume managed by a volume plugin into the container.
+
+        `options` can be a comma-delimited list of:
+           + `[ro|rw]` to mount a volume read-only or read-write, respectively. By default, the volumes are mounted read-write.
+           + `[z|Z]` The z option tells Docker that two containers share the volume content. As a result, Docker labels the content with a shared content label. Shared volume labels allow all containers to read/write content. The Z option tells Docker to label the content with a private unshared label. Only the current container can use a private volume. Labeling systems like SELinux require that proper labels are placed on volume content mounted into a container. Without a label, the security system might prevent the processes running inside the container from using the content. By default, Docker does not change the labels set by the OS.
```

Comment:
Make sure you label `z` and `Z` with backticks everywhere they occur. I'd use a bullet for each of them if you are not going to turn the whole thing into a table. Combining them might lead skim-readers to think that they both do the same thing.